src/test/java/testresults/CVSB_980/AutoCalculatedTestResults_2695.java

```diff
@@ -0,0 +1,92 @@
+package testresults.CVSB_980;
+
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import pages.TestPage;
+import steps.*;
+import steps.composed.TestTypeCategoryComp;
+import util.BaseTestClass;
+
+import java.util.HashMap;
+
+@RunWith(SerenityRunner.class)
+public class AutoCalculatedTestResults_2695 extends BaseTestClass {
+
+    @Steps
+    TestSteps testSteps;
+
+    @Steps
+    TestTypeCategorySteps testTypeCategorySteps;
+
+    @Steps
+    TestTypeCategoryComp testTypeCategoryComp;
+
+    @Steps
+    EUVehicleCategorySteps euVehicleCategorySteps;
+
+    @Steps
+    OdometerReadingSteps odometerReadingSteps;
+
+    @Steps
+    TestReviewSteps testReviewSteps;
+
+    @Steps
+    TestTypeDetailsSteps testTypeDetailsSteps;
+
+    @Steps
+    DefectCategorySteps defectCategorySteps;
+
+    @Steps
+    DefectItemSteps defectItemSteps;
+
+    @Steps
+    DefectDescriptionSteps defectDescriptionSteps;
+
+    @Steps
+    DefectDetailsSteps defectDetailsSteps;
+
+    @Steps
+    SeatbeltInstallationCheckSteps seatbeltInstallationCheckSteps;
+
+    @Steps
+    AdvisoryDetailsSteps advisoryDetailsSteps;
+
+    @Title("CVSB_980 - AC1 - Pass criteria (Minor + Advisory defects)")
+    @Test
+    public void passCriteriaWithNoDefects() {
+        testTypeCategoryComp.goToTestPage("TC7524","Avello Edinburgh Ltd");
+        testSteps.selectVehicleCategoryOption();
+        euVehicleCategorySteps.selectM1Option();
+        testSteps.selectOdometerReading();
+        odometerReadingSteps.typeInField("8");
+        odometerReadingSteps.pressSave();
+        testSteps.addTestType();
+        testTypeCategorySteps.selectFromTestTypeList("Annual test");
+        testSteps.selectTestType("Annual test", TestPage.TestTypeStatuses.IN_PROGRESS);
+        testTypeDetailsSteps.setCarriedOutDuringTest(true);
+        testTypeDetailsSteps.selectNumberOfSeatbeltsFitted();
+        seatbeltInstallationCheckSteps.inputNumberOfSeatbelts("8");
+        testTypeDetailsSteps.pressSave();
+        testSteps.reviewAction();
+        testReviewSteps.checkTestStatus("Annual test", "PASS");
+        testReviewSteps.changeDetails();
+        testTypeDetailsSteps.clickAddDefect();
+        defectCategorySteps.searchForDefect("59");
+        defectCategorySteps.selectDefectFromList("59. Brake Systems and Components");
+        defectItemSteps.selectDefectFromList("4. Reservoir:");
+        defectDescriptionSteps.tapAddAnAdvisoryNote();
+        advisoryDetailsSteps.addCustomNoteAndTapAddNote("Test");
+        defectDescriptionSteps.selectDefect("59.4 (a) (i) MINOR");
+        HashMap<String, String> map = new HashMap<>();
+        map.put("Vertical", "Upper");
+        map.put("Lateral", "Nearside");
+        map.put("Longitudinal", "Front");
+        map.put("Axle Number", "8");
```

Comment:
why use hashmap in tests? please encapsulate in steps/page

Reply:
It is just a parameter like a string and I think this approach is better than giving a method 8 string arguments and encapsulating the logic into the step pages because it will be more complicated to construct the map this way rather than just instantiate it in the actual test. This explication is valid also for the below comments regarding similar issues.